### ISSUE TITLE
[EN] Stuck in critial error loop

### ISSUE THREAD

**Issue by MomoSinX** [Atlantiss/BugTracker#2432]
I was flying around in Borean Tundra to get the map explored, when this error happened. No idea what causes it but every time I log-in I get it so I am pretty much stuck in this loop. I made a screenshot, maybe it helps.. If someone could teleport me to Stormwind that would be great (char name is Raishin). Tried dismounting and falling to my death but apparently that's impossible because after a few sec I get the crash. Didn't see any unstuck option on the site.

![fatal error](https://cloud.githubusercontent.com/assets/11248873/6427839/0c70e498-bf8d-11e4-94d4-3df7d670add3.jpg)

**Comment by Daknyas** [comment]
Log-in with another character, then try again. It worked for me.

**Comment by MomoSinX** [comment]
Didn't solve it :/.

**Comment by Daknyas** [comment]
Try to set your graphic settings to the lowest possible then if you have DX11, set it to 9. However this error may have many causes...

**Comment by MomoSinX** [comment]
That didn't solve it either. I think it might be server side, some object can't load or something.

**Comment by Daknyas** [comment]
Yeah, most probably you're right. I had this problem and I solved it in that way, so I dunno what else you can try. :/ Wait for someone more competent.

**Comment by Daknyas** [comment]
Oh wait, there's an unstuck function in the site, but you need 5 vote points to use it. Go to your vote panel, select the character you want to unstuck, go to Accont and character (at the left), then untuck.
Hope it helps!

**Comment by MomoSinX** [comment]
Ohhh, man, that unstuck option was hidden really well. Anyway, solved my problem, thanks a lot! Gonna avoid that problematic area from now on.

**Comment by Dziara** [comment]
I also encountered that place before unstuck option was available. I downloaded mini client once again and flew away from that place before all textures loaded on that client :P

**Comment by Tj20** [comment]
I've flown some circles around Warsong Hold but could not reproduce this error, everything works fine for me.

I've noticed that there is an elevator nearby and I think those are problematic right now, but I think they don't crash the client.

![pic](http://i.imgur.com/XP6OrCi.jpg)

**Comment by tdotzlod** [comment]
Got this error in same area on 3 toons.
Tried to delete cache, bring other toon (that's how all 3 got stuck), re-downloading entire client, disabling addons, changing configuration for client and nothing helped.
This error havent happened before (was lvling in borean couple weeks ago and had no problem)

**Comment by Tj20** [comment]
Tried this again today: I've stayed in the area about 30 minutes because I thought it might happen only every couple of minutes (like the crashes in Kaskala because of the fighting mobs), but I didn't experience any crashes here.

**Comment by aje** [comment]
can you give the full error message please
